src/lab3new/Lab3new.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package lab3new;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author class
+ */
+public class Lab3new {

```

Comment:
From the lab instructions:
> create a new Java project named Lab3 and a new class called `Calculator`.
As such, the name of this class should be `Calculator`.